src/js/contentscripts/dnt.js

```diff
@@ -20,28 +20,34 @@ function getPageScript() {
   // code below is not a content script: no chrome.* APIs /////////////////////
 
   // return a string
-  return "(" + function (NAVIGATOR, OBJECT) {
+  return "(" + function () {
 
     if (NAVIGATOR.doNotTrack != "1") {
       OBJECT.defineProperty(OBJECT.getPrototypeOf(NAVIGATOR), "doNotTrack", {
         get: function doNotTrack() {
           return "1";
-        }
+        },
+        configurable: true,
+        enumerable: true
       });
     }
 
-    if (!NAVIGATOR.globalPrivacyControl) {
+    if (!'globalPrivacyControl' in NAVIGATOR.prototype) {
```

Comment:
Do you want to override this signal when present by the browser? If so, I would change this check as such:

```suggestion
    if (!globalThis.navigator.globalPrivacyControl) {
```

Reply:
I think we should, consistently with what we do with DNT, attempt to set GPC if it isn't set to true already.